dev-support/ranger-docker/Dockerfile.ranger-usersync

```diff
@@ -14,9 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG RANGER_BASE_IMAGE
-ARG RANGER_BASE_VERSION
-FROM ${RANGER_BASE_IMAGE}:${RANGER_BASE_VERSION}
+FROM ranger
```

Comment:
Why is it necessary to change the base image (`FROM @{RANGER_BASE_IMAGE}`) to `ranger`?

`ranger` is the image containing Ranger admin server; usersync shouldn't use `ranger` as the base.